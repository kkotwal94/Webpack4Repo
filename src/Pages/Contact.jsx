import React, { useState, useMemo } from 'react';

const Contact = () => {
  const [number, setNumber] = useState(0);
  return (
    <div>
      Contact page #{number}
      <button
        onClick={() => {
          setNumber(number + 1);
        }}
      >
        Increment +
      </button>
    </div>
  );
};

export default Contact;
