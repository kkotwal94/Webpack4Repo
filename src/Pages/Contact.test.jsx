import React from 'react';
import Contact from './Contact';
import renderer from 'react-test-renderer';

test('Testing Contact updating', () => {
  const component = renderer.create(<Contact />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
