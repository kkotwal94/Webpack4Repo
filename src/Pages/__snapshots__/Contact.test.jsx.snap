// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Contact updating 1`] = `
<div>
  Contact page #
  0
  <button
    onClick={[Function]}
  >
    Increment +
  </button>
</div>
`;
